packages/j/jodit.json

```diff
@@ -21,7 +21,8 @@
       {
         "basePath": "build",
         "files": [
-          "jodit*.+(css|js)"
+          "jodit*.+(css|js)",
+          "**/*"
```

Comment:
👋 We generally try to avoid using open globstars in file maps as it can lead to files being matched unintentionally that shouldn't be on the CDN, causing performance impact to our pipelines and making the library responses on the site/API confusing for users.

Could this be scoped to just what you intend to match? Looking at your repo, I suspect this might just need to be `plugins/**/*.@(css|js)`?

Reply:
Hi. Thank you. Corrected. Just did not understand about @, is this symbol definitely needed?

Reply:
`@(css|js)` means `css or js` in glob syntax :)

Reply:
You can play around with different glob patterns at https://www.digitalocean.com/community/tools/glob if you want

Reply:
Thank you. Returned the @ sign)